packages/react/src/components/checkbox/use-checkbox-group.ts

```diff
@@ -87,17 +76,15 @@ export const useCheckboxGroup = <
 
   const getCheckboxProps: PropGetter<
     { value?: Y },
-    { checked?: boolean; isChecked?: boolean; value?: Y }
+    { checked?: boolean; value?: Y }
   > = useCallback(
     (props = {}, ref = null) => ({
       ...props,
       ref,
-      [isNative ? "checked" : "isChecked"]: value.some(
-        (val) => String(props.value) === String(val),
-      ),
+      checked: value.some((val) => String(props.value) === String(val)),
```

Comment:
Since we are removing `isNative` prop here, I am assuming `checked` is used only by default

Reply:
I think it is true, too.